man/mblaze.7

```diff
@@ -3,81 +3,81 @@
 .Os
 .Sh NAME
 .Nm mblaze
-.Nd introduction to mblaze
+.Nd introduction to the mblaze message system
 .Sh DESCRIPTION
 The
 .Nm
-message system is a set of Unix utilities to deal with
-mail kept in Maildir folders.
+message system is a set of Unix utilities for processing and
+interacting with mail messages which are stored in maildir folders.
 .Pp
 Its design is roughly inspired by MH, the RAND Message Handling
 System, but it is a complete implementation from scratch.
 .Pp
 .Nm
-consists of these Unix tools that each do one job:
+consists of these Unix utilities that each do one job:
+.Pp
 .Bl -tag -width 11n -compact
 .It Xr maddr 1
-extract addresses from mail
+extract mail addresses from messages
 .It Xr magrep 1
-find mails matching a pattern
+print messages matching a pattern
 .It Xr mbnc 1
-bounces mail
+bounce messages
 .It Xr mcom 1
-compose and send mail
+compose and send messages
 .It Xr mdeliver 1
-deliver messages or import mailboxes
+deliver messages or import mbox file
 .It Xr mdirs 1
-find Maildir folders
+list maildir folders, recursively
 .It Xr mexport 1
-export Maildir folders as mailboxes
+export messages as mbox file
 .It Xr mflag 1
-change flags (marks) of mail
+manipulate maildir flags
 .It Xr mflow 1
-reflow format=flowed plain text mails
+reflow format=flowed plain text messages
 .It Xr mfwd 1
-forward mail
+forward messages
 .It Xr mgenmid 1
-generate Message-IDs
+generate a Message-ID
 .It Xr mhdr 1
-extract mail headers
+print message headers
 .It Xr minc 1
-incorporate new mail
+incorporate new messages
 .It Xr mless 1
-conveniently read mail in
+conveniently read messages in
 .Xr less 1
 .It Xr mlist 1
-list and filter mail messages
+list and filter messages
 .It Xr mmime 1
-create MIME messages
+encode MIME messages
 .It Xr mmkdir 1
-create new Maildir
+create new maildir folders
 .It Xr mpick 1
-advanced mail filter
+advanced message filter
 .It Xr mrep 1
-reply to mail
+reply to messages
 .It Xr mscan 1
-generate one-line summaries of mail
+generate one-line message summaries
 .It Xr msed 1
-manipulate mail headers
+manipulate message headers
 .It Xr mseq 1
-manipulate mail sequences
+manipulate message sequences
 .It Xr mshow 1
-render mail and extract attachments
+render messages and extract MIME parts
 .It Xr msort 1
-sort mail
+sort messages
 .It Xr mthread 1
-arrange mail into discussions
+arrange messages into discussions
 .El
-.Sh PRINCIPLES
+.Pp
 .Nm
 is a classic command line MUA and has no features
-for receiving or transferring mail;
-you are expected to fetch your mail using
+for receiving or transferring messages;
+you are expected to fetch your messages using
 .Xr fdm 1 ,
-.Xr getmail 1
+.Xr getmail 1 ,
 .Xr offlineimap 1 ,
-.Xr procmail 1 ,
-or similar ,
+or similar utilities,
```

Comment:
perhaps we could mention here that having a Maildir spool is ok too of course?

Reply:
You mean like a local maildir in `/var/spool/mail` or somesuch? IIRC it can sometimes be a bit of a mess to set one up (mbox is fine) and OpenBSD actively discourages it. Or what do you have in mind?

Reply:
I mean that the MDA directly delivers into `~/Maildir` (e.g. like qmail or postfix do).

Reply:
I'll try out some wording for this, and might try to use it as on opportunity to get rid of / incorporate in a different way the 'expects your mail to reside in maildir folders' line.